github-setup.md

```diff
@@ -77,4 +77,4 @@ Once our secret manager is setup, we want to **restart our terminal/WSL Shell**.
 4. Now that we have our token, we can use it when git prompts us for a password. If you are on Windows, you won't receive a password prompt but instead you will receive a pop-up that asks for you Personal Access Token. Because we previously configured our password/secret manager, our input will be saved for us. This way, we don't need to keep track of our PAT.
 
 ### Still having trouble?
-We can make use of another form of authentication, however you may run into some unanticipated issues depending on your network setup. If you'd like to explore this option, please checkout [this guide](https://hackmd.io/cZQaolzKS0iEK9bkJYSWgA)
+We can make use of another form of authentication, however you may run into some unanticipated issues depending on your network setup. GitHub supports SSH authentication. SSH stands for Secure SHell and is an older asynchronous cryptography system of authentication that generates private identifier data and public identifier data. The private data stays on your local machine and is validated against the public part of the record to verify your identity. This provides a way of securely operating network services over an unsecured network(such as the internet at large). If you'd like to explore this option, please checkout [this guide](https://hackmd.io/@AgDXdHgSSPKsJIhCxlaTuA/BJtNu88fF)
```

Comment:
You can keep this short and sweet. "If you have trouble with this setup or a preference for ssh you can follow [this guide](link)."